### PR TITLE
Making the nav

### PR DESCRIPTION
**Pull request by AlexJCarmichael**
Open to the idea of adding an icon next to the SpaceBook text. Also completely flexible on background color.

### CODE REVIEW

**Comment by VanAnderson** [comment]
I merged this locally and tested it, it looks pretty good! The only thing is that the 'events' button goes to '/invites'. Was this intentional? I thought we had discussed at one point that invites will not actually have an index page but rather be inserted into other views.

**Comment by AlexJCarmichael** [comment]
Our thoughts on an index page for invites would let users have a chance to see any open events. The index will have pending and attended events. Each of the three pages will have an option to make an invite.

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class DashboardController < ApplicationController
 
   def home
-    @user = "Danediah Auflorescence GoblinSlayer III"
```
Comment (vpatel90) [inline app/controllers/dashboard_controller.rb:4]: I approve

**Comment by vpatel90** [comment]
👍

**Pull request merged**